requirements/test.txt

```diff
@@ -13,7 +13,7 @@ attrs==22.2.0
     #   -r requirements/base.txt
     #   openedx-events
     #   pytest
-avro==1.10.0
+avro==1.11.1
```

Comment:
There's a change that starts validating enum defaults in Python, but I don't think we even support those yet: https://issues.apache.org/jira/browse/AVRO-3229